treeprint.go

```diff
@@ -159,6 +169,24 @@ func (n *node) SetMetaValue(meta MetaValue) {
 	n.Meta = meta
 }
 
+func (n *node) VisitAll(fn NodeVisitor) {
+	for _, node := range n.Nodes {
+		// if len(node.Nodes) == 0 {
```

Comment:
Left over debugging code?

Reply:
Well, not really. My idea here was to give an example on how to determine if the item you're iterating is a branch or a node. Should I take the comments out and add them as normal text in the docs?

Reply:
Scratch all that. I was looking at the wrong place. Yeah it was just me testing. :facepalm: . Removed it.